src/client/components/EventDepartments.js

```diff
@@ -45,12 +45,13 @@ const DepartmentList = ({ departments }) => (
     <h3 className={theme.title}>
       <FormattedMessage id="departments.title" defaultMessage="Departments" />
       <div className={style.departmentList}>
-        { departments.map(department => {
+        { departments.map((department, i) => {
```

Comment:
woo lack of type safety :'-(
is there any way of avoiding this kind of problem?